rustfmt.toml

```diff
@@ -0,0 +1 @@
+fn_args_layout = "Compressed"
```

Comment:
What's this for?

Reply:
https://rust-lang.github.io/rustfmt/?version=v1.4.36&search=fn_args_layout

Reply:
Dosn't seem to work though, so I added rustfmt::skip
If I don't I get this when running `cargo fmt`:
```rust
#[instrument]
#[allow(clippy::type_complexity)]
pub fn get_rev_dependency(
    db: &Connection, crate_name: &str, expected_crate_dep_name: &str,
) -> Result<
    Vec<
        Result<
            (
                String,
                String,
                String,
                String,
                Result<Vec<(String, String)>, cratesio_dbdump_csvtab::rusqlite::Error>,
            ),
            cratesio_dbdump_csvtab::rusqlite::Error,
        >,
    >,
    cratesio_dbdump_csvtab::rusqlite::Error,
> {
```